ldap/servers/plugins/pwdstorage/gost_yescrypt.c

```diff
@@ -62,3 +63,25 @@ gost_yescrypt_pw_enc(const char *pwd)
 
     return enc;
 }
+
+#else
+
+/*
+ * We do not have xcrypt, so always fail all checks.
+ */
+int
+gost_yescrypt_pw_cmp(const char *userpwd __attribute__((unused)), const char *dbpwd __attribute__((unused)))
+{
+    slapi_log_err(SLAPI_LOG_ERR, GOST_YESCRYPT_SCHEME_NAME,
+                  "Unable to use GOST_YESCRYPT, xcrypt is not available.\n");
```

Comment:
It's really more of a nitpick, - it looks to me that the error message will look like 

```
"ERR - GOST_YESCRYPT - Unable to use GOST_YESCRYPT, xcrypt is not available."
```

As a thought, may be it makes sense to use the function name `'gost_yescrypt_pw_cmp'` instead of `GOST_YESCRYPT_SCHEME_NAME` so we can avoid redundancy?

Reply:
Yes, this is a good point.